src/ngx_http_lua_kong_module.c

```diff
@@ -407,42 +431,78 @@ ngx_http_lua_kong_cleanup(void *data)
         sk_X509_pop_free(ctx->upstream_client_certificate_chain, X509_free);
         EVP_PKEY_free(ctx->upstream_client_private_key);
     }
+
+    if (ctx->upstream_trusted_store != NULL) {
```

Comment:
We should rewrite this function to use the `ngx_http_lua_kong_cleanup_cert_and_key` and `ngx_http_lua_kong_cleanup_trusted_store` helper instead.